docs/specs/toc.yml

```diff
@@ -814,3 +814,86 @@ outputs:
   build.log: |
     ["warning","file-not-found","Cannot find file 'index.md' relative to 'docs/f1/TOC.yml'","docs/f1/TOC.yml"]
     ["error","invalid-toc-href","The toc href 'index.md' can only reference to a local TOC file, folder or absolute path","docs/f1/TOC.yml"]
+---
+# Individual experimental TOC should be built
+inputs:
+  docfx.yml:
+  docs/f1/TOC.experimental.md: |
+    # [Index Reference](../index.md)
+  docs/f2/TOC.experimental.yml: |
```

Comment:
>TOC.experimental.yml [](start = 10, length = 20)

add experimental metadata for testing

Reply:
I will support experimental metadata in another PR

---
In reply to: [206505749](https://github.com/dotnet/docfx/pull/3127#discussion_r206505749) [](ancestors = 206505749)